docs/how_to_guide/19_posterior_parameters.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "898a1f9b",
+   "metadata": {},
+   "source": [
+    "# How to use `PosteriorParameters` in sbi\n",
+    "\n",
+    "The `PosteriorParameters` classes in SBI are dataclasses used to initialize parameters for different types of posterior algorithms, when using the `build_posterior` method. This guide shows how to configure and use PosteriorParameters in your workflow.\n",
+    "\n",
+    "`sbi` currently implements the following PosteriorParameters subclasses. Each subclass of `PosteriorParameters` holds parameters for a specific inference method. \n",
+    "\n",
+    "- `DirectPosteriorParameters`: used with DirectPosterior\n",
+    "\n",
+    "- `ImportanceSamplingPosteriorParameters`: used with ImportanceSamplingPosterior\n",
+    "\n",
+    "- `MCMCPosteriorParameters`: used with MCMCPosterior\n",
+    "\n",
+    "- `RejectionPosteriorParameters`: used with RejectionPosterior\n",
+    "\n",
+    "- `VectorFieldPosteriorParameters`: : used with VectorFieldPosterior\n",
+    "\n",
+    "- `VIPosteriorParameters`: used with VIPosterior"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c4df01a",
+   "metadata": {},
+   "source": [
+    "# Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7c6d86e",
+   "metadata": {},
+   "source": [
+    "The example below uses `MCMCPosteriorParameters` for passing parameter values. You can follow a similar approach as the example below with your `sbi` workflow when using the other `PosteriorParameters` types."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5c1d2e5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "\n",
+    "from sbi.inference import NRE\n",
+    "from sbi.utils.torchutils import BoxUniform\n",
+    "\n",
+    "\n",
+    "def simulator(theta):\n",
+    "    return 1.0 + theta + torch.randn(theta.shape, device=theta.device) * 0.1\n",
+    "\n",
+    "num_dim = 3\n",
+    "prior = BoxUniform(low=-2 * torch.ones(num_dim), high=2 * torch.ones(num_dim))\n",
+    "theta = prior.sample((300,))\n",
+    "x = simulator(theta)\n",
+    "\n",
+    "inference = NRE(prior=prior)\n",
+    "inference.append_simulations(theta, x)\n",
+    "inference.train(max_num_epochs=1)"
```

Comment:
please add a comment here that we are setting this to 1 for speed purposes. in practice, this should not be set to 1.